config/initializers/mailer.rb

```diff
@@ -0,0 +1,2 @@
+require Rails.root.join("lib", "notify_delivery_method")
```

Comment:
We can just use `require "notify_delivery_method"` I believe here, since lib is in the load path.